src/data/family.py

```diff
@@ -297,6 +297,10 @@ def screens_by_uniq_id(self) -> dict[str, Screen]:
                 screens_by_uniq_id[screen.uniq_id] = screen
         return screens_by_uniq_id
 
+    @cached_property
+    def calculated_first_screen_id(self) -> str | None:
+        return next(iter(self.screens_by_uniq_id.keys()))
```

Comment:
```suggestion
    @cached_property
    def calculated_first_screen_id(self) -> str | None:
        return next(iter(self.screens_by_uniq_id.keys()), None)
```
Ironic that that's what we've talked about today ^^ next without default return value raises a StopIteration error when the generator is empty. You can use it without default value when the generator is never empty, but the typing and the usage of the property suggests you need a default None value